src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -129,13 +125,6 @@ def validate_key_import_source(ns):
         raise ValueError('--pem-password must be used with --pem-file')
 
 
-def validate_key_ops(ns):
-    allowed = [x.value.lower() for x in JsonWebKeyOperation]
-    for p in ns.key_ops or []:
-        if p not in allowed:
-            raise ValueError("unrecognized key operation '{}'".format(p))
```

Comment:
Removing this method removes all validation couldn't you just change the condition to

 if p.lower() not in allowed:

since you've already put all the values to lower?

Reply:
NM seems I see from your comments that enum_choice_list seems to handle this validation

Reply:
Yes, enum_choice_list handles this for us. Not sure why I implemented it this way unless the KV SDK at the time didn't have enums.